lib/rollbar.rb

```diff
@@ -796,14 +796,7 @@ def require_core_extensions
     end
 
     def monkey_patch_socket?
-      return false unless defined?(ActiveSupport::VERSION::STRING)
```

Comment:
Testing failed with active support 4.2. So just monkey patch always active support is loaded.